### ISSUE TITLE
Why we call Mix-up method as data augmentation technique?

### ISSUE THREAD

**Issue by Nafees-060** [facebookresearch/mixup-cifar10#25]
I am confused about the Mixup data augmentation technique, let me explain the problem briefly:
We double or quadruple the data using classic augmentation techniques (e.g., Jittering, Scaling, Magnitude Warping). For instance, if the original data set contained 4000 samples, there will be 8000 samples in the data set after the augmentation.

On the other hand, according to my understanding, in Mixup data augmentation, we do not add the data but rather mix the samples and their labels and use these new mixed samples for training to produce a more regularized model. Am I correct? If yes, then why is the Mixup method referred to as data augmentation? Since we only mix samples and not artificially increase the data set size?

**Comment by michaelku1** [comment]
I had the same question too, and my guess is that it's because the purpose of their experiments is to reflect the effectiveness of training with the goal of vicinal risk minimization. They basically try to prove that VRM is better than empirial risk minimization by showing the test errors of various models trained with VRM and ERM. In order to better reflect the difference, it'd better to train on original data (empirical) vs virtual data (vicinal) for the experiment to validate their theory. However, it'd be interested to see how virtual examples perform along with the original training data, but they did not do that since it's not the focus of the experiment.

Now, they did mention in the paper that it is a form data augmentation because of what it causes the training examples to behave, but again the paper does not say their goal is to compare the performance between model trained on original data with that trained on original + virtual data. However, that may be a good  experiment to perform for further analysis. Therefore, I will tell you is that feel free to use that as an actual data augmentation technique to augment your examples besides the fact that they did not.

<img width="845" alt="Screenshot 2022-04-01 at 19 01 23" src="https://user-images.githubusercontent.com/48415065/161251235-b1dd5cef-59b1-40f9-8571-46aac5733c95.png">
.
